Cabal/Distribution/PackageDescription/Parsec.hs

```diff
@@ -363,6 +364,10 @@ parseName pos args = case args of
          parseFailure pos $ "Invalid name " ++ show args
          pure ""
 
+parseUnqualComponentName :: Position -> [SectionArg Position] -> ParseResult UnqualComponentName
+parseUnqualComponentName pos args = mkUnqualComponentName <$> parseName pos args
```

Comment:
I wonder if `parseName` is used anywhere else, otherwise we could inline it here.